spec.bs

```diff
@@ -1933,33 +2056,39 @@ SKOS constructs such as <code>skos:narrower</code> <em class="rfc2119">MAY</em>
 Definitions {#definitions}
 =====================
 The following terms are used throughout this specification:
+  * <dfn>Agent</dfn> -- a distinct individual, group, organization, or piece of
+        software with an identity that can be strongly authenticated
+  * <dfn>Arc</dfn> -- the directed relationship between two nodes in an RDF graph
   * <dfn>Body Graph</dfn> -- the RDF graph derived from parsing the body of an
         HTTP request with a parser determined by the Content-type: header of that
         HTTP request.
-  * <dfn>Container</dfn> -- The generalized notion of a collection of resources;
+  * <dfn>Container</dfn> -- the generalized notion of a collection of resources;
         implementations of [=shape trees=] <em class="rfc2119">MAY</em> use a
         container implementation such as [[LDP]]
         (ldp:Container, ldp:BasicContainer, etc.)
   * <dfn>Ecosystem</dfn> -- a software environment with resources organized in
         some hierarchical grouping that rely on [=shape tree=] concepts to
         better organize and validate structures of data
-  * <dfn>Focus Node</dfn> -- The IRI to an RDF subject which is used to
+  * <dfn>Focus Node</dfn> -- the IRI to an RDF subject which is used to
         specify which subject within a document should be processed first.  In
-        the context of [=shape trees=] one usage is directing shape validation
+        the context of [=shape trees=] one usage is directing [=shape=] validation
         to the appropriate node in the graph.
   * <dfn>Managed Container</dfn> -- any [=container=] that has a [=shape tree=]
         planted in it. A [=Managed Container=] <em class="rfc2119">MAY</em>
         be an [=Instance Root=] or hierarchically nested within the
         resource hierarchy.
-  * <dfn>Non-RDF Source</dfn> -- The generalized notion of document not
+  * <dfn>Non-RDF Source</dfn> -- the generalized notion of document not
         containing linked-data triples; this may include
         plain text or binary data.
   * <dfn>RDF</dfn> -- Resource Description Framework [[rdf11-primer]]
-  * <dfn>Resource</dfn> -- The generalized notion of document containing linked-data;
+  * <dfn>Resource</dfn> -- the generalized notion of document containing linked-data;
         implementations of [=shape trees=] may use a resource implementation such
         as [[LDP]] (ldp:Resource, etc.)
-  * <dfn>Shape Tree Data Instance</dfn>A resource hierarchy that adheres to one
-         or more [=shape trees=]
+  * <dfn>Shape</dfn> -- a schema definition allowing validation of an RDF
+        subject.  See [this](https://www.w3.org/TR/shacl/#dfn-shape) for more
+        details.
```

Comment:
The general shape description here is good, but rather than linking to the SHACL specific definition of a shape, i think it is better to reference the ShEx and SHACL specifications as supported shape specifications in the second part of this definition

Reply:
Updating to." a schema definition allowing validation of an RDF
        subject.  Example specifications supporting the notion of shapes include 
        [[ShEx]] and [[SHACL]]."